src/Project.WebMVC/Areas/Identity/Pages/Account/Register.cshtml.cs

```diff
@@ -40,7 +41,7 @@ public class InputModel
             
             [Required]
             [DataType(DataType.Date)]
-            public DateTime Birthday { get; set; }
+            public LocalDate Birthday { get; set; }
```

Comment:
Why did you switch to the 3rd party type?

Reply:
I want to calculte user's age in current timezone context, but I just had a thought and made a conlusion. I should store the time in UTC on the server side and work with timezones on the client side, so my previous solution is wrong

Reply:
It seems like user's bithday shouldn't have to depend on curent timezone  👍

Reply:
I will try to do smth similar to LocalDate in NodaTime with DateTime to remove dependencies

Reply:
Done